tests/rules/no-extra-arguments.test.ts

```diff
@@ -125,11 +126,31 @@ ruleTester.run('no-extra-arguments', rule, {
         {
           message: encodedMessage(0, 1, [
             { line: 4, column: 18, endLine: 4, endColumn: 26, message: 'Formal parameters' },
+            { line: 2, column: 12, endLine: 2, endColumn: 13, message: 'Extra argument' },
           ]),
         },
       ],
       options: ['sonar-runtime'],
     },
+    {
```

Comment:
why those 2 new tests?

Reply:
The contributor probably wanted to demonstrate that the rule also works with the special `arguments` object. Since it's unrelated to the ticket, I will drop those.